streams/src/main/java/org/apache/kafka/streams/processor/internals/assignment/AssignorConfiguration.java

```diff
@@ -347,22 +348,28 @@ public AssignmentListener assignmentListener() {
         public final long probingRebalanceIntervalMs;
```

Comment:
I think this comment got lost in a discussion thread, but can we add a note to AssignorConfiguration pointing out that any Streams configs added here will need to be explicitly passed through? It seems like it's too easy to fall into this same trap again

Reply:
Ah, right. Sure thing!